src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/MetadataDecoder.vb

```diff
@@ -204,6 +204,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
                     End If
 
                     Do
+                        If symbol.IsTupleType Then
+                            Return IsOrClosedOverATypeFromAssemblies(symbol.TupleUnderlyingType, assemblies)
```

Comment:
This code appears to be correct and precisely parallel to the C# implementation, so I'll leave it as it stands.